diff --git a/extensions/typescript-language-features/src/utils/versionProvider.ts b/extensions/typescript-language-features/src/utils/versionProvider.ts
index eb50009cfd..c3877beb06 100644
--- a/extensions/typescript-language-features/src/utils/versionProvider.ts
+++ b/extensions/typescript-language-features/src/utils/versionProvider.ts
@@ -140,7 +140,7 @@ export class TypeScriptVersionProvider {
 	public get bundledVersion(): TypeScriptVersion {
 		try {
 			const { extensionPath } = vscode.extensions.getExtension('vscode.typescript-language-features')!;
-			const typescriptPath = path.join(extensionPath, '../node_modules/typescript/lib');
+			const typescriptPath = path.join(extensionPath, './node_modules/typescript/lib');
 			const bundledVersion = new TypeScriptVersion(typescriptPath, '');
 			if (bundledVersion.isValid) {
 				return bundledVersion;
diff --git a/extensions/typescript-language-features/package.json b/extensions/typescript-language-features/package.json
index a5147d60f8..8639051b21 100644
--- a/extensions/typescript-language-features/package.json
+++ b/extensions/typescript-language-features/package.json
@@ -2,7 +2,7 @@
   "name": "typescript-language-features",
   "description": "%description%",
   "displayName": "%displayName%",
-  "version": "1.0.0",
+  "version": "1.35.1",
   "icon": "icon.png",
   "author": "vscode",
   "publisher": "vscode",
diff --git a/extensions/typescript-language-features/src/utils/logDirectoryProvider.ts b/extensions/typescript-language-features/src/utils/logDirectoryProvider.ts
index af6886e704..c0b924c56b 100644
--- a/extensions/typescript-language-features/src/utils/logDirectoryProvider.ts
+++ b/extensions/typescript-language-features/src/utils/logDirectoryProvider.ts
@@ -30,7 +30,7 @@ export default class LogDirectoryProvider {
 		try {
 			const path = this.context.logPath;
 			if (!fs.existsSync(path)) {
-				fs.mkdirSync(path);
+				fs.mkdirSync(path, {recursive: true});
 			}
 			return this.context.logPath;
 		} catch {
